### ISSUE TITLE
Missing plotmark asterisk

### ISSUE THREAD

**Issue by lucaventurini** [nschloe/matplotlib2tikz#151]
When specifying marker asterisk, something does not work.

Python code:
```python
import matplotlib.pyplot as plt
plt.plot(1,1,marker="*")
```

Output is:
```
file encoding: None
=========================================================
Please add the following lines to your LaTeX preamble:

\usepackage[utf8]{inputenc}
\usepackage{fontspec} % This line only for XeLaTeX and LuaLaTeX
\usepackage{pgfplots}
\usepgfplotslibrary{plotmarks}
=========================================================
```

and

```latex
% This file was created by matplotlib2tikz v0.6.3.
\begin{tikzpicture}

\definecolor{color0}{rgb}{0.12156862745098,0.466666666666667,0.705882352941177}

\begin{axis}[
xmin=0.945, xmax=1.055,
ymin=0.945, ymax=1.055,
tick align=outside,
x grid style={lightgray!92.026143790849673!black},
y grid style={lightgray!92.026143790849673!black}
]
\addplot [semithick, color0, mark=asterisk*, mark size=3, mark options={solid}]
table {%
1 1
};
\end{axis}

\end{tikzpicture}
```

I suspect that `\usepgfplotslibrary{plotmarks}` should be `\usetikzlibrary{plotmarks}` instead, since the first does not exist, but even with this manual correction the `mark=asterisk*` does not produce anything.

**Comment by janbrumm** [comment]
I just ran into the same problem. The points is that pgfplots doesn't know a marker `asterisk*` but only `asterisk`. I think I located the problem in /matplotlib2tikz/line2d.py within function `_mpl_marker2pgfp_marker()` starting from line 254: 
```python
        if (
            marker_face_color is not None
            and (
                not isinstance(marker_face_color, str)
                or marker_face_color.lower() != "none"
            )
            and pgfplots_marker not in ["|", "-"]
        ):
            pgfplots_marker += "*"
```
This should read:
```python
        if (
            marker_face_color is not None
            and (
                not isinstance(marker_face_color, str)
                or marker_face_color.lower() != "none"
            )
            and pgfplots_marker not in ["|", "-", "asterisk", "star"]
        ):
            pgfplots_marker += "*"
```
As both `asterisk` and `star` are only available without `*`.

**Comment by nschloe** [comment]
Fixed by #259.
